exp/views/user.py

```diff
@@ -186,9 +184,10 @@ class ResearcherDetailView(LoginRequiredMixin, DjangoPermissionRequiredMixin, ge
     def get_queryset(self):
         """
         Restrict queryset so org admins can only modify users in their organization
+        or unaffiliated researchers.
         """
         qs = super().get_queryset()
-        return qs.filter(organization=self.request.user.organization)
+        return qs.filter(Q(Q(organization=self.request.user.organization) | Q(is_researcher=True, groups__isnull=True, organization__isnull=True)))
```

Comment:
I think this needs a distinct() at the end like Researcher List View.  Get errors sometimes depending on the user `MultipleObjectsReturned at /exp/researchers/2/
get() returned more than one User -- it returned 6!`